changelog.d/expire_internal_metrics.breaking.md

```diff
@@ -0,0 +1,4 @@
+The global `expire_metrics_secs` configuration option now defaults to 300s rather than being
+disabled. Set to a negative value to disable expiration.
```

Comment:
> Set to a negative value to disable expiration.
Maybe worth adding to the note, that setting this to a negative value will preserve the existing (old) behavior.

Reply:
Good call 👍